### ISSUE TITLE
workspace validate does not work

### ISSUE THREAD

**Issue by bertsky** [OCR-D/core#175]
If i try
```ocrd workspace -d ws1 validate```
this leads to 
> OSError: Error reading file 'ws1/None': failed to load external entity "ws1/None"

And passing a parameter with the mets URL is not allowed either:
```ocrd workspace validate ws1/mets.xml```
gives me
> Error: Got unexpected extra argument (ws1/mets.xml)

So how am I supposed to validate a workspace?

**Comment by bertsky** [comment]
Please reopen. There still seems to be a bug luring in here. I do pass pytest test/validator/test_workspace_validator.py, but I cannot get it to work on the command line:

```sh
cd assets/data
ocrd workspace clone -a blumenbach_anatomie_1805/mets.xml ws1
ocrd workspace validate ws1/mets.xml
```

This fails with `TypeError: cannot parse from 'NoneType'` and an unhelpful stacktrace of not being able to parse _some_ XML document. Adding a line 
```diff
--- a/ocrd/validator/workspace_validator.py
+++ b/ocrd/validator/workspace_validator.py
@@ -123,5 +123,6 @@ class WorkspaceValidator(object):
 
     def _validate_page(self):
         for ocrd_file in self.mets.find_files(mimetype=MIMETYPE_PAGE, local_only=True):
+            log.debug('page file url=%s ID=%s fileGrp=%s', ocrd_file.url, ocrd_file.ID, ocrd_file.fileGrp)
             page_report = PageValidator.validate(ocrd_file=ocrd_file, strictness=self.page_strictness)
             self.report.merge_report(page_report)
```

at least gives me valuable logging output when debugging. The file in question is the very first PAGE file in the workspace, and its filesystem URL _is_ valid, albeit with a `file://` prefix. 

It appears that it is that very prefix which `PageValidator.validate` via `ocrd_page.from_file` via `ocrd_page_generateds.parse` is not swallowing.

**Comment by bertsky** [comment]
Fixed in 1.0, thanks!
